load/LoadOrganizations.py

```diff
@@ -120,8 +122,10 @@ def process_other_agreement(self, language_record, type_code, fileset_id):
 					licensors.append(language_record.Licensor())
 				else:
 					print(f"ERROR Agreement Type is 'Other' and Methodology 'VirtualRecording' but licensor is empty for fileset id: {fileset_id}")
+			elif type_code == "audio":
+				licensors.append(self.LicensorHosanna)
 			else:
-				print(f"ERROR Agreement Type is 'Other', Methodology is 'VirtualRecording', but type is not text. Unable to assign for fileset id: {fileset_id}")
+				print(f"ERROR Agreement Type is 'Other', Methodology is 'VirtualRecording', but type is not text or audio. Unable to assign for fileset id: {fileset_id}")
```

Comment:
actually, we can remove this log statement, because we handle video separately. So all cases should be handled and we typically shouldn't emit any logs for Virtual Recording

Reply:
@bradflood , I understand. This log has been removed as you can see here: https://github.com/faithcomesbyhearing/dbp-etl/pull/136/files#diff-0628aedffa821d1b9acf4c81d50c756fe72674fcc7ca34c47e0a5b0a65c60aceL124